x-pack/test/alerting_api_integration/spaces_only/tests/alerting/migrations.ts

```diff
@@ -398,14 +398,9 @@ export default function createGetTests({ getService }: FtrProviderContext) {
 
       // Only the rule that was enabled should be tagged
       expect(responseEnabledBeforeMigration.body._source?.alert?.tags).to.eql([
-        '__internal_rule_id:064e3fed-6328-416b-bb85-c08265088f41',
-        '__internal_immutable:false',
         'auto_disabled_8.0',
       ]);
-      expect(responseDisabledBeforeMigration.body._source?.alert?.tags).to.eql([
```

Comment:
Just dropping a conservative note; This change makes me a little nervous - if we were to incidentally collect non-prebuilt rule telemetry we would be in breach of our privacy agreement with users. I trust you have done your due diligence and testing on the security telemetry instrumentation. Before merging can you please confirm that the following queries aren't adversly affected by this change?

https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts#L431-L477

and

https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts#L510-L631

Reply:
I can see these queries are not relying on internal tags, but on `immutable` parameter.
These removed immutable tag is in fact derived from `immutable` parameter. So, removal of internal tag won't affect queries above